core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -500,19 +507,34 @@ class TestCaseWriter {
                                     (value is Map<*, *>) -> handleMapLines(test_index, value, lines)
                                     (value is String) -> longArray = true
                                     else -> {
-                                        val printableTh = handleFieldValues(value)
-                                        if (printableTh != "null"
-                                                && printableTh != NOT_COVERED_YET
-                                                && !printableTh.contains("logged")
-                                                && !printableTh.contains("""\w+:\d{4,5}""".toRegex())
+                                        val printableFieldValue = handleFieldValues(value)
+                                        if (printableFieldValue != "null"
```

Comment:
it looks like you are doing exactly the same check on a string value 3 times, with also copied&pasted comment. this should be refactored in a private method that takes as input a string, and return a boolean